flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -95,38 +82,35 @@
 
 /** IT cases for all cassandra sinks. */
 @SuppressWarnings("serial")
-@Category(FailsOnJava11.class)
-// this test is known to be unstable, but the exact cause is unknown
-@RetryOnException(times = 2, exception = NoHostAvailableException.class)
 public class CassandraConnectorITCase
         extends WriteAheadSinkTestBase<
                 Tuple3<String, Integer, Integer>,
                 CassandraTupleWriteAheadSink<Tuple3<String, Integer, Integer>>> {
 
-    private static final Logger LOG = LoggerFactory.getLogger(CassandraConnectorITCase.class);
-
-    @Rule public final RetryRule retryRule = new RetryRule();
-
-    private static final boolean EMBEDDED = true;
+    private static final String IMAGE_TAG = "3.0";
 
-    private static EmbeddedCassandraService cassandra;
+    @ClassRule
+    public static final CassandraContainer CASSANDRA_CONTAINER =
+            createCassandraContainer(IMAGE_TAG);
```

Comment:
Well, we should also reference the version in the `DockerImageVersions` ;)

Reply:
ah; actually we should remove IMAGE_TAG; it is effectively unused.

Reply:
oops, leftover